webpack/assets/javascripts/react_app/components/common/EmptyLine/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Skeleton from 'react-loading-skeleton';
+import { translate as __ } from '../../../../react_app/common/I18n';
+
+const EmptyLine = ({ isLoading }) => {
+  if (isLoading) {
+    return <Skeleton />;
```

Comment:
How about override `Skeleton` props by the caller?

Reply:
Added a prop for that